api/config/config_preprod.json

```diff
@@ -47,8 +47,8 @@
     "operationalTerraWorkspaceName": "aouwgscohortextractionpreprod",
     "operationalTerraWorkspaceBucket": "fc-5d8e6afe-9429-44ae-a22a-67b09705d1b4",
     "extractionPetServiceAccount": "pet-110393474898566940625@aouwgscohortextraction-preprod.iam.gserviceaccount.com",
-    "extractionMethodConfigurationNamespace": "aouwgscohortextraction-preprod",
-    "extractionMethodConfigurationName": "",
+    "extractionMethodConfigurationNamespace": "aouwgsextraction-manual-testing",
```

Comment:
I'd rather not connect preprod to anything that's named this experimentally. The workflow will be operating on production data, and I'd prefer to name anything that interacts with it accordingly. Was there an issue with the previous namespace?